revampedClient/__tests__/components/cards/Card.test.js

```diff
@@ -0,0 +1,19 @@
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import Card from '../../../components/cards/Card';
+
+describe('Card component', () => {
+  it('should render correctly', () => {
+    const props = {
+      height: '10px',
```

Comment:
the required props for the Card component is style and children, why add height and width as props in your test

Reply:
yeah I forgot about that when I made the previous change lol. thanks